rmgpy/molecule/group.py

```diff
@@ -542,9 +542,14 @@ class GroupBond(Edge):
     group if it matches *any* item in the list.
     """
 
-    def __init__(self, atom1, atom2, order=None):
+    def __init__(self, atom1, atom2, order=[]):
```

Comment:
I recall it being bad practice to set default values to mutable types, but I'm not sure how applicable that is in this particular case. It might be worth considering to switch back to None though.